packages/shared/src/settings/index.ts

```diff
@@ -141,3 +152,13 @@ export const useScopedSettings = (
 
   return scopeSettings(settings, scopes);
 };
+
+export const useScopedSettings = (
+  scopes: ScopeDefinition
+): ScopedSettingsReturn => {
+  return useMemo(() => {
+    return getScopedSettings(scopes);
+  }, [scopes]);
```

Comment:
usememo might not work as expected here ... the scopes object can change without its reference changing, for example

```
const scopes = { organization, experiment };

delete scopes.experiment;

// scopes is still the same reference; useMemo will not re-calculate
```

Reply:
removed the hook entirely based on our conversation